lib/ex_aws/dynamo.ex

```diff
@@ -596,6 +591,103 @@ defmodule ExAws.Dynamo do
     request(:delete_item, data)
   end
 
+  @doc """
+  Update item in table
```

Comment:
@kyleaa Is this the correct @doc info for `transact_get_items`? Perhaps

"Initialize a transaction to get an item or items from one or multiple tables"

Reply:
These might not really match with the style of the docs written here so far, but this is an excerpt from the AWS docs that I think sum it up relatively well:

"a synchronous operation that atomically retrieves multiple items from one or more tables (but not from indexes) in a single account and Region"

Reply:
Cool, thanks.